lisa/features/nvme.py

```diff
@@ -21,7 +21,7 @@ class Nvme(Feature):
     _device_pattern = re.compile(r".*(?P<device_name>/dev/nvme[0-9]$)", re.MULTILINE)
     # brw-rw---- 1 root disk 259, 0 Jun 21 03:08 /dev/nvme0n1
     _namespace_pattern = re.compile(
-        r".*(?P<namespace>/dev/nvme[0-9]n[0-9]$)", re.MULTILINE
+        r".*(?P<namespace>/dev/nvme[0-9]n[0-9]+$)", re.MULTILINE
```

Comment:
Add new example above.

Reply:
This change is to support nvme namespaces more than 9.
Example added.